library/std/src/os/unix/net/ancillary.rs

```diff
@@ -270,6 +360,7 @@ impl SocketCred {
     }
 
     /// Get the current PID.
+    #[must_use]
```

Comment:
Similarly, why put `#[must_use]` here, but not on `get_pid` or `get_uid`?

Reply:
This change is in the netbsd implementation. Also it's missing from the other `get_*` in the netbsd implementation.